cylc/flow/network/schema.py

```diff
@@ -1211,9 +1211,10 @@ async def mutator(root, info, command=None, workflows=None,
         workflows = []
     if exworkflows is None:
         exworkflows = []
-    w_args = {}
-    w_args['workflows'] = [Tokens(w_id) for w_id in workflows]
-    w_args['exworkflows'] = [Tokens(w_id) for w_id in exworkflows]
+    w_args = {
+        'workflows': [Tokens(w_id) for w_id in workflows],
+        'exworkflows': [Tokens(w_id) for w_id in exworkflows]
+    }
```

Comment:
This change is for a new flake8-simplify rule SIM904 "Initialize dictionary directly".

Reply:
(Now duplicated as a separate PR: #4684 )